tests/testSSMmixins.py

```diff
@@ -0,0 +1,276 @@
+from __future__ import with_statement
+import unittest
+import os
+import numpy as np
+
+import lsst.utils.tests as utilsTests
+from lsst.utils import getPackageDir
+from lsst.sims.utils import ObservationMetaData
+from lsst.sims.catalogs.generation.db import fileDBObject
+from lsst.sims.catalogs.measures.instance import InstanceCatalog, compound
+from lsst.sims.catUtils.mixins import PhotometrySSM, AstrometrySSM
+from lsst.sims.photUtils import BandpassDict, SedList, PhotometricParameters
+from lsst.sims.coordUtils import _observedFromICRS
+
+class LSST_SSM_photCat(InstanceCatalog, PhotometrySSM):
+    column_outputs = ['id', 'lsst_u' ,'lsst_g', 'lsst_r', 'lsst_i', 'lsst_z', 'lsst_y']
+
```

Comment:
I like these unit tests as they're really helpful for me to understand how to use the new classes and mixins.